### ISSUE TITLE
Is it possible to list all the jobs on the map?

### ISSUE THREAD

**Issue by jerome281** [gediyaj/jobify#2]
Hi,

I thought one setting would be really nice to have if possible to display all the active jobs on the map?

I've notice right now only reading the post on the homepage reading the data-latitude  & data-longitude

`MarkersCollectionView.prototype.parseResults = function(event) {
        var data, html;
        if (!_.isUndefined(event && !_.isUndefined(event.target))) {
          html = $(event.target).find(this.map.settings.list).first().find('li');
        } else {
          html = $(this.map.settings.list).first().find('li');
        }
        data = _.map(html, function(i) {
          return $(i).data();
        });
        return data = _.filter(data, function(i) {
          return _.has(i, 'latitude');
        });
      };`

Any tips to accomplish this?

Many thanks,

**Comment by gediyaj** [comment]
Yes It is possible to list jobs on map. You can see example here : https://sandbox-jobify-demos.astoundify.com/classic/find-a-job/?search_keywords=&search_location=&search_categories%5B%5D=17&submit=Search&search_region=&filter_job_type%5B%5D=

Let me know if you need any help from me.

Thanks
Jignesh
